internal/metrics/prometheus.go

```diff
@@ -79,6 +87,19 @@ const (
 	MetricNameConfigPushDuration         = "ingress_controller_configuration_push_duration_milliseconds"
 )
 
+// Fallback config push metrics names.
+const (
+	MetricNameFallbackTranslationCount           = "ingress_controller_fallback_translation_count"
+	MetricNameFallbackTranslationBrokenResources = "ingress_controller_fallback_translation_broken_resource_count"
+	MetricNameFallbackConfigPushCount            = "ingress_controller_fallback_configuration_push_count"
+	MetricNameFallbackConfigPushSuccessTime      = "ingress_controller_fallback_configuration_push_last"
+	MetricNameFallbackConfigPushDuration         = "ingress_controller_fallback_configuration_push_duration_milliseconds"
+	MetricNameFallbackConfigPushBrokenResources  = "ingress_controller_fallback_configuration_push_broken_resource_count"
+	MetricNameFallbackCacheGeneratingDuration    = "ingress_controller_fallback_cache_generating_duration_milliseconds"
```

Comment:
Wouldn't this be clearer to call it: 

```suggestion
	MetricNameFallbackCacheGenerationDuration    = "ingress_controller_fallback_cache_generation_duration_milliseconds"
```
?